src/Activity.js

```diff
@@ -102,10 +102,7 @@ class Activity {
     let data = this.activityData;
     let sortedUserArray = (userRepo.makeSortedUserArray(id, data)).reverse();
     let streaks = sortedUserArray.reduce((streakDays, currentDay, index) => {
-      let qualifyingDay;
-      if (index >= 2) {
-        qualifyingDay = (sortedUserArray[index - 2][relevantData] < sortedUserArray[index - 1][relevantData] && sortedUserArray[index - 1][relevantData] < sortedUserArray[index][relevantData]);
-      };
+      let qualifyingDay = this.isQualifyingDay(sortedUserArray, index, relevantData);
```

Comment:
Glad we got some clarification in the check-in and separated this complicated logic out - this seems friendlier at least!